### PR TITLE
If you are running the python script, what is the version in bug_report.yaml

### PR DESCRIPTION
**Pull request by KendallDoesCoding**
# Related Issue/Addition to code

- If people are running the python script. some versions may not allow this program to run (old versions) so I think its better to add a field of fill in the python version

## Type of change

- [ ] This change requires a documentation update

# Proposed Changes
- Added field of fill in the python version.

### CODE REVIEW

**Comment by ThioJoe** [comment]
It already asks for version

**Comment by KendallDoesCoding** [comment]
> It already asks for version

I thought that was the release version of this project? This asks for the python version.

**Comment by rachmadaniHaryono** [comment]
based on poetry, it require minimum python 3.6 but remember python 3.6 security support ended last december

https://endoflife.date/python

https://github.com/ThioJoe/YT-Spammer-Purge/pull/459

**Comment by Firecul** [comment]
> It already asks for version

It asks for tool version, not python version

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 1]
```diff
@@ -30,11 +30,17 @@ body:
...
+    attributes:
+      label: Python version
+      required: false
+      description: If you are running the Python script (not the exe) please mention the version of Python you are running. If you are running the exe file, please ignore this.)
   - type: input
     id: version
     attributes:
       label: Release version
-      description: What version of this program are you using? (can be found in the title when running (eg. ======== YOUTUBE SPAMMER PURGE VERSION (insert version here) or in the `version` variable in the `YouTubeSpammerPurge.py` file.)
+      description: What version of this program are you using? (can be found in the title when running (eg. ======== YOUTUBE SPAMMER PURGE VERSION (insert version here) or in the `version` variable in the `YouTubeSpammerPurge.py` file.)
```
Comment (Firecul) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:43]: I see those trailing spaces 😉

**Pull request closed without merging**